Add governance to "development team" (#22)
ESMValGroup/ESMValTool-website: team.md

title: Development team
---

## Governance

> The [ESMValTool governance can be found here](/assets/pdf/ESMValTool_Governance.pdf). A consortium agreement is

Reviewer: ```suggestion
The [ESMValTool governance structure is described here](/assets/pdf/ESMValTool_Governance.pdf). A consortium agreement is
```
